src/llir/abi.rs

```diff
@@ -115,10 +115,10 @@ impl ArgEncoding {
         impl fmt::Display for Impl<'_> {
             fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
                 match &self.0 {
-                    Enc::Integer { arg0: true, ty_color, size, immediate, format } => write!(
+                    Enc::Integer { arg0: true, ty_color, size, immediate, format, extend } => write!(
```

Comment:
Agreed, I would've definitely changed that earlier if I'd realized `..` could be referenced instead of needing to explicitly list everything just to be able to refer to it. Guess that's another thing to add to the growing list of "cool things I didn't think rust would allow".

Reply:
That's not what it's doing here, the `..` is totally ignoring them/not binding them to any names.

Reply:
never mind, this change can't be made since it's an enum